package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@terrestris/mapfish-print-manager",
-  "version": "6.0.0",
+  "version": "6.1.0",
```

Comment:
Is this intended?

Reply:
Yes, since https://github.com/terrestris/mapfish-print-manager/releases/tag/v6.1.0 is the latest release. Maybe something got broken during the last release process where a version for `ol5` branch has been published as well.
I've added this to keep tagged / released versions in sync.